package.json

```diff
@@ -5,7 +5,7 @@
   "dependencies": {
     "@types/jasmine": "^2.8.9",
     "decoders": "^1.10.6",
-    "elm": "^0.19.0-bugfix2",
+    "events": "^3.0.0",
```

Comment:
I do have a concern about the events package, which is this in the docs for `on` (emphasis mine):

> Adds the listener function to the end of the listeners array for the event named eventName. **No checks are made to see if the listener has already been added. Multiple calls passing the same combination of eventName and listener will result in the listener being added, and called, multiple times.**

It don't know why node opted to create a non-idempotent event API, but it seems a bit crazy to me. It makes it much harder to work with as a client. It's also not really expected in the browser space where [element event listeners behave differently](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener#Multiple_identical_event_listeners):

> If multiple identical EventListeners are registered on the same EventTarget with the same parameters, the duplicate instances are discarded. They do not cause the EventListener to be called twice, and they do not need to be removed manually with the